test/test-rotateElement.js

```diff
@@ -0,0 +1,91 @@
+var assert = require('chai').assert;
+var tetris = require('../tetris')
+describe("element rotation around the axis", function () {
+    var elementTetromino = tetris.getCoordinates(`
+		---
+		--X
+		---
+	`)
+
+    it("element move up one from the right", function () {
+        let stepElementTetromino = tetris.rotateElement(elementTetromino)
```

Comment:
This function should accept 2 parameters: coorditates of coordinate to rotate and "pivot" around which rotation is performed

Reply:
Please apply to all invocations

Reply:
Done